src/ceph-disk/ceph_disk/main.py

```diff
@@ -1448,6 +1448,7 @@ def mount(
 
 def unmount(
     path,
+    dont_rm=False,
```

Comment:
@shun-s i'd suggest rename this parameter to `do_rm`, it would be easier to understand if we could use positive logic.